CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## Unreleased
+
+* Change the way we generate pretty print data for our documentations ([PR #2934](https://github.com/alphagov/govuk_publishing_components/pull/2934))
```

Comment:
Nitpick: can this match the PR title? (or vice versa)

Reply:
done